dashboard/app/mailers/pd/workshop_mailer.rb

```diff
@@ -103,7 +103,7 @@ def exit_survey(workshop, teacher, enrollment)
     mail content_type: content_type,
       from: from_hadi,
       subject: 'How was your Code.org workshop?',
-      to: email_address(@teacher.name, @teacher.email)
+      to: email_address(@enrollment.name || @teacher.name, @teacher.email)
```

Comment:
We talked about being more explicit about how we ask for their display name going forward, asking for first and last name for teachers, iirc.  Did we decide to reuse the display name field for that work?  @tanyaparker might have thoughts here.  Ideally we're heading towards a comprehensive single identity per teacher.